docs/src/user-manual/ingest/submitting-content.md

```diff
@@ -112,23 +112,81 @@ package exceeds the configured limit.
    at the top of the browse results. Each SIP uploaded will be ingested via its
    own separate workflow.
 
-## Add SIPs via a source location
+## Initiate ingest using SIPs uploaded to a source location
 
-Coming soon!
+Another method of initiating ingest is by selecting packages previously uploaded
+to a [source location]. A **SIP source location** is an object storage location
+or filesystem directory for the deposit of SIPs that can then be ingested by an
+Enduro operator. The [SIP source configuration] must be set up by a system
+administrator.Unlike a [watched location] where an action is automatically
+triggered upon deposit, a source location is intended for asynchronous manual
+follow-up. In this case, Enduro operators can select zipped packages found in a
+configured source location and initiate ingest via the user interface.
+
+The deposit or upload process will depend on the source location used - to see
+an example of how packages are uploaded to a [MinIO] object store, see the
+watched location [example shown below](#example---upload-via-minio).
+
+**To initiate ingest using SIPs in a source location**:
+
+1. Using the [navbar](../overview.md#navigation) on the left side of the screen,
+   click on "Upload SIPs." Enduro will redirect you to the SIP upload  page.
+
+    ![The upload SIPs page](../screenshots/local-upload.png)
+
+2. For SIP ingest from a source location, click the "Select from source" tab at
+   the top of the page. Enduro will display a picker interface showing any
+   zipped packages that have been added to the configured source location
+
+    !!! warning
+
+        SIPs **must be zipped** to be properly ingested into Enduro. You cannot
+        start an ingest workflow from an unzipped directory.
+
+        SIPs _may_ be placed in subdirectories, but these subdirectories will
+        only be visible as part of the filepath of the ZIP - see for example the
+        last SIP shown in the screenshot below, which is in a `my-stuff`
+        folder in the configured source location. This means that any
+        subdirectories added to the source location that do **not** have one or
+        more zipped SIPs added to them will also not be shown in the source
```

Comment:
Any files will cause a subdirectory to be displayed, not just zipped SIPs.

Reply:
ok, tried to fix  - thoughts welcome